Documentation/Signoffs/WorkSysten(1).md

```diff
@@ -0,0 +1,163 @@
+
+- This is proven within the calculations for velocity
+
+
+### Meeting the Sensing Speed Constraint
+
+The Maximum Velocity that the Work System supports should be no higher than 190 miles per hour to account for the fastest speed achieved on a bike being 186 miles per hour [5]. The minimum speed that needs to be supported is 0 miles per hour. This will allow users to reach very small speeds without having to worry that the Kit will not record the data to the Work System. 
+
+The Maximum Velocity that the Work System can record at is well above the 190 MPH value set by the constraint. The minimum velocity that the kit can support is no velocity at all and can support extremely small changes in velocity as long as the interval between rotations does not exceed the battery life of the system.
+
+- Given the slowest delay within the system is 10 microseconds and the minimum wheel diameter that needs to be supported is 20 Inches we can use the following formulas.
+  
+  **Circumference = Diameter * Pi**
+  
+  **Velocity = Distance / Time**
+
+- Using these formulas we get circumference is 62.83 and maximum velocity given 10 microseconds is 6283000 inches per second or 356989 miles per hour.
+
+
+### Meeting the Splash Resistant Constraint
+
+A box shall house the work system. The components inside the bike along with the box will be coated in a splash-resistant silicone to prevent water damage from occurring. This is in the case that the rider is to use the Ride Replay Kit in the rain or ride through a body of water that could potentially splash the work system or fully submerge it. The electronics will be coated in an acrylic conformal coating that will be done using a 419D Liquid that protects from moisture and is certified IPC-CC-830B [6]. To seal the box to add a second layer of protection an acrylic-based sealant will be used and conforms to the military specification MIL-46106 which provides a strong sealant and water repellent.
+
+### Meeting the Ride Time Constraint
+
+Since there are many different trail lengths and skill levels, there is no set amount of duration for trails. Due to this the minimum amount of time that the batteries should last is hard to make a specific number for. Team 5 decided to set the minimum amount of time at 24 hours. This allows users to record large trails without having to worry about the batteries not lasting the entire trail. 
+
+- To solve for the maximum time the battery can last, the amperage output for each device and the milliamp hours output of the batteries is needed. The following values are needed: the current draw for the Zigbee Transmitter, the current draw for the Hall effect generator, and the current output of the battery. The values are 33 milliamps, 10 milliamps, and a minimum value for modern AA batteries is 2000 milliamp hours respectively. The following formulas can be used:
+
+**Amphours = Amps * Hour**
+
+**Hours = Amp Hours / Amps**

```

Comment:
This is accurate only if the voltage required is equal for all devices and the battery. Is this the case? Also, AAs tend to drop voltage as they discharge. So, if this is a problem, the entire battery life won't be usable.